src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Native lazy-loading for the web
+subhead: Browser-level native lazy-loading is finally here!
+authors:
+  - houssein
+  - addyosmani
+  - mathiasbynens
+date: 2019-07-17
+hero: hero.png
+alt: Phone outline with loading image and assets
+description: |
+  This post covers the loading attribute and how it can be used
+  to control the loading of images and iframes.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
```

Comment:
```suggestion
  - performance
draft: true
```

Reply:
Hey, @housseindjirdeh, can you accept this change so we can merge this post while we're waiting for it to be ready to publish?

Reply:
I think Houssein is OOO so I'm going to merge this and then add the draft flag myself.